packages/yew/src/dom_bundle/braw.rs

```diff
@@ -0,0 +1,220 @@
+                    next_sibling = node.clone().unchecked_ref::<Element>().next_sibling();
+                    parent
+                        .remove_child(&node)
+                        .expect("failed to remove braw node");
+                }
+            }
+        }
+    }
+}
+
+impl ReconcileTarget for BRaw {
+    fn detach(self, _root: &BSubtree, parent: &Element, _parent_to_detach: bool) {
+        self.detach_bundle(parent);
+    }
+
+    fn shift(&self, next_parent: &Element, next_sibling: NodeRef) -> NodeRef {
+        if let Some(node) = self.reference.cast::<Element>() {
+            if let Some(parent) = node.parent_node() {
+                parent.remove_child(&node).unwrap();
+            }
+
+            next_parent
+                .insert_before(&node, next_sibling.get().as_ref())
+                .unwrap();
+
+            return NodeRef::new(node.into());
+        }
+        NodeRef::default()
+    }
+}
+
+impl Reconcilable for VRaw {
+    type Bundle = BRaw;
+
+    fn attach(
+        self,
+        _root: &BSubtree,
+        _parent_scope: &AnyScope,
+        parent: &Element,
+        next_sibling: NodeRef,
+    ) -> (NodeRef, Self::Bundle) {
+        let elements = BRaw::create_elements(&self.html);
+        if elements.is_empty() {
+            return (
+                next_sibling.clone(),
+                BRaw {
+                    reference: next_sibling,
+                    children_count: 0,
+                },
+            );
+        }
+        let node_ref = NodeRef::default();
+
+        let count = elements.len();
+        let mut iter = elements.into_iter();
+        let first = iter.next().unwrap();
+        insert_node(&first, parent, next_sibling.get().as_ref());
+        node_ref.set(Some(first.into()));
+        for child in iter {
+            insert_node(&child, parent, next_sibling.get().as_ref());
+        }
+        (
+            node_ref.clone(),
+            BRaw {
+                reference: node_ref,
+                children_count: count,
+            },
+        )
+    }
+
+    fn reconcile_node(
+        self,
+        root: &BSubtree,
+        parent_scope: &AnyScope,
+        parent: &Element,
+        next_sibling: NodeRef,
+        bundle: &mut BNode,
+    ) -> NodeRef {
+        // we don't have a way to diff what's changed in the string so we remove the node if it's
+        // present and reattach it
+        if let BNode::Raw(raw) = bundle {
+            raw.detach_bundle(parent)
+        }
+        let (node_ref, braw) = self.attach(root, parent_scope, parent, next_sibling);
+        *bundle = braw.into();
+        node_ref
+    }
+
+    fn reconcile(
+        self,
+        root: &BSubtree,
+        parent_scope: &AnyScope,
+        parent: &Element,
+        next_sibling: NodeRef,
+        bundle: &mut Self::Bundle,
+    ) -> NodeRef {
+        // we don't have a way to diff what's changed in the string so we remove the node and
+        // reattach it
```

Comment:
This fact may also be a bit surprising to the user. Can we at least keep the elements if the `self.html` is the same as before (save that in the BRaw struct)? I think otherwise, this may lead to problems like shifting focus, and a lot of other dom interactions on spurious re-renders.